seeds/_sector/dex/smardex_polygon_base_trades_seed.csv

```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+polygon,smardex,1,2024-06-11,0x885b2833498c911aca89a8c0219686583538c75706795b25c478cba709756418,476,0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,58014407,3290,3503000228043981967
+polygon,smardex,1,2024-06-11,0xddc565236cbf833c0e56404002cd93ee2c2435f642c0ea1c00d0d48046eb3e84,170,0x111111517e4929d3dcbdfa7cce55d30d4b6bc4d6,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,58015096,226786000000000000,581594
```

Comment:
Hi @blanchemaria6, please verify seed records, seems token_bought and token_sold are flipped:
https://dune.com/queries/3821590

Reply:
I fixed the seed file and it works now @Hosuke